modules/clickhouse/clickhouse_test.go

```diff
@@ -218,22 +218,35 @@ func TestClickHouseWithConfigFile(t *testing.T) {
 }
 
 func performCRUD(conn driver.Conn) ([]Test, error) {
-	err := conn.Exec(context.Background(), "create table if not exists test_table (id UInt64) engine = MergeTree PRIMARY KEY (id) ORDER BY (id) SETTINGS index_granularity = 8192;")
-	if err != nil {
-		return nil, err
-	}
+	var (
+		maxRetry = 5
+		delay    = 100 * time.Millisecond
+		err      error
+		rows     []Test
+	)
 
-	err = conn.Exec(context.Background(), "INSERT INTO test_table (id) VALUES (1);")
-	if err != nil {
-		return nil, err
-	}
+	for i := 0; i < maxRetry; i++ {
```

Comment:
Given we already use backoff package, let's use it here too. Not a blocker, though

Reply:
ok, on it

Reply:
updated @mdelapenya

Reply:
Thanks! The CI is running. Will approve it after the CI passes, thanks!